src/CommunityToolkit.Aspire.Hosting.Supabase/README.md

```diff
@@ -0,0 +1,147 @@
+
+## Core API
+
+### AddSupabase
+
+```csharp
+public static IResourceBuilder<SupabaseResource> AddSupabase(
+    this IDistributedApplicationBuilder builder,
+    string name,
+    IResourceBuilder<ParameterResource>? password = null,
+    int? apiPort = null,
+    int? dbPort = null);
+```
+
+- **name**: The logical name of the Supabase resource.
+- **password**: Optional `ParameterResource` builder for the Postgres password; default is a random-generated parameter.
+- **apiPort**: Host port to bind the Supabase HTTP API (default 8000).
+- **dbPort**: Host port to bind the Postgres database (default 5432).
+
+This registers:
+1. A `SupabaseResource` container running Postgres and HTTP API images
+2. Environment variables for `POSTGRES_USER`, `POSTGRES_DB`, `POSTGRES_PASSWORD`, and standard `PG*` variables
+3. Volume binds for:
+   - `./volumes/db/data` → `/var/lib/postgresql/data`
+   - `./volumes/db/migrations` → `/docker-entrypoint-initdb.d/migrations`
+   - `./volumes/db/init-scripts` → `/docker-entrypoint-initdb.d/init-scripts`
+4. Two endpoints on the resource:
+   - `http`: API port
+   - `postgresql`: Database port
+
+### AddAllSupabase
+
+```csharp
+public static IResourceBuilder<SupabaseResource> AddAllSupabase(
+    this IDistributedApplicationBuilder builder,
+    string name,
+    IResourceBuilder<ParameterResource>? password = null,
+    int? apiPort = null,
+    int? dbPort = null);
+```
+
+Chains `AddSupabase(...)` and then adds all child services via extension methods:
+- `WithKong()`
+- `WithStudio()`
+- `WithRest()`
+- `WithRealtime()`
+- `WithStorageService()`
+- `WithAuthService()`
+- `WithMetaService()`
+- `WithInbucketService()`
+- `WithImageProxyService()`
+- `WithLogflareService()`
+- `WithEdgeRuntimeService()`
+
+## Child Modules
+
+Each child service is added with a `.WithXXX()` method. For example:
+
+```csharp
+supabase.WithAuthService();
+```
+
+These methods launch the corresponding container images with default configuration and link them to the core Supabase resource.
+
+## Customizing Data & Migrations
+
+Place your SQL migrations under:
+```
+./volumes/db/migrations
+```
+
+Any additional init scripts (roles, extensions, JWT setup) go under:
+```
+./volumes/db/init-scripts
+```
+
+Persistent database data is stored in:
+```
+./volumes/db/data
+```
+
+## SupabaseResource
+
+The `SupabaseResource` exposes two endpoints:
+
+- **PrimaryEndpoint** (`http`) – for HTTP API calls
+- **DatabaseEndpoint** (`postgresql`) – for Postgres connections
+
+Use the provided `ConnectionStringExpression` to wire up downstream resources:
+
+```csharp
+builder.AddProject<MyApp>("api")
+       .WithReference(supabase)
+       .WithDbContext<MyDbContext>((sp, options) =>
+           options.UseNpgsql(
+               supabase.ConnectionStringExpression.GetValueAsync().Result));
```

Comment:
??? Why is this in the hosting integration, this isn't what any other database does. Also Task.Result....no please

Reply:
This was just AI slop when i asked it to reformat the README.md to a more readable markdown i will do a proof read when i convert from draft to a normal PR.
Thanks catching it